prof/src/com/oracle/souffleprof/ProgramRun.java

```diff
@@ -522,9 +522,9 @@ public Relation getRelation(String name) {
         for (int i = 0; i < table.length; i++) {
             for (int j = 0; j < table[0].length; j++) {
                 if (table[i][j] == null) continue;
-                if (table[i][j].getClass() == Double.class) {
+                if (table[i][j] instanceof Double) {
                     new_table[i][j] = formatTime((Double)table[i][j]);
-                } else if (table[i][j].getClass() == Long.class) {
+                } else if (table[i][j] instanceof Long) {
                     new_table[i][j] = formatNum(precision, (Long)table[i][j]);
                 } else {
                     new_table[i][j] = table[i][j];
```

Comment:
If the assumption is that it's going to be a String here, it might be good to add an explicit test or cast so that it will error out on anything unexpected.